src/Cart/CartClient.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Cart;
+
+use ArrowSphere\PublicApiClient\AbstractClient;
+use ArrowSphere\PublicApiClient\Exception\NotFoundException;
+use ArrowSphere\PublicApiClient\Exception\PublicApiClientException;
+use GuzzleHttp\Exception\GuzzleException;
+
+class CartClient extends AbstractClient
+{
+    /**
+     * @var string The base path of the Cart API
+     */
+    private const ROOT_PATH = '/cart';
+
+    /**
+     * @var string The base path of the API
+     */
+    protected $basePath = self::ROOT_PATH;
```

Comment:
You can override this property in the constructor, the same way as you did it in  `SpportClient`